integration/qbft/scenarios/integration_test_type.go

```diff
@@ -36,12 +37,20 @@ import (
 
 // IntegrationTest defines an integration test.
 type IntegrationTest struct {
-	Name              string
-	OperatorIDs       []spectypes.OperatorID
-	InitialInstances  map[spectypes.OperatorID][]*protocolstorage.StoredInstance
-	Duties            map[spectypes.OperatorID][]*spectypes.Duty
-	ExpectedInstances map[spectypes.OperatorID][]*protocolstorage.StoredInstance
-	StartDutyErrors   map[spectypes.OperatorID]error
+	Name               string
+	OperatorIDs        []spectypes.OperatorID
+	Identifier         spectypes.MessageID
+	ValidatorDelays    map[spectypes.OperatorID]time.Duration
+	InitialInstances   map[spectypes.OperatorID][]*protocolstorage.StoredInstance
+	Duties             map[spectypes.OperatorID][]scheduledDuty
+	ExpectedInstances  map[spectypes.OperatorID][]*protocolstorage.StoredInstance
+	InstanceValidators map[spectypes.OperatorID][]func(*protocolstorage.StoredInstance) error
```

Comment:
Are we keeping both `ExpectedInstances` and `InstanceValidators`?

Reply:
@amir-blox `ExpectedInstances` is used in round change and f+1 decided, `InstanceValidators` is used in regular. This is an intermediate version to be able to run all tests and make sure they all pass. Afterwards, I will get rid of `ExpectedInstances`